ovssubnet/controller/kube/kube.go

```diff
@@ -72,6 +91,44 @@ func (c *FlowController) manageLocalIpam(ipnet *net.IPNet) error {
 	return nil
 }
 
+func (c *FlowController) SetupIptables(fw *firewalld.Interface, containerNetwork string) error {
+	if fw.IsRunning() {
```

Comment:
Consider making this block more data-driven:

	for _, x := range []struct {
		ipv   string
		table string
		chain string
		value int
		args  []string
	}{
		{firewalld.IPv4, "nat", "POSTROUTING", 0, []string{"-s", ...}}
	} {
		err := fw.EnsureRule(x.ipv, x.table, x.chain, x.value, x.args)
		if err != nil {
			return err
		}
	}